staging/src/k8s.io/apimachinery/pkg/labels/selector_test.go

```diff
@@ -630,3 +630,78 @@ func BenchmarkSelectorFromValidatedSet(b *testing.B) {
 		}
 	}
 }
+
+func TestRequiresExactMatch(t *testing.T) {
+	testCases := []struct {
+		name  string
+		sel   Selector
+		label string
+		found bool
+		value string
+	}{
+		{
+			"keyInOperatorExactMatch",
```

Comment:
Can you please explicitly add
"fieldName": "valueName"
in all tests (e.g. name: "keyInOp...")

It's much easier to maintain (e.g. when you want to add a new field to the struct to extend tests).

Reply:
fine. fixed it